src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos: todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
```

Comment:
Avoid using one-letter variable names like `todo`. Consider using a more descriptive name to improve code readability, as per the checklist item: 'Don't use one-letter variable naming.'